### ISSUE TITLE
Icon request: folder-arrow-right/folder-arrow-left

### ISSUE THREAD

**Issue by ricardo17coelho** [FortAwesome/Font-Awesome#20013]
### Use case

There are already the variations:
- folder-arrow-up
- folder-arrow-down

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] This icon does not represent a brand
- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The request is for a concrete object, or I've included an example image
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it can already be made with the new Font Awesome Wizard! [Take a look at how it works.](https://blog.fontawesome.com/icon-wizard/)

**Comment by ricardo17coelho** [comment]
@sensibleworld 
Well, i understand that this "could be done" with Wizard.. but is not the same icon.
![Image](https://github.com/user-attachments/assets/1acafc96-4d03-4e32-ae69-36d2c8d3a051)

![Image](https://github.com/user-attachments/assets/9663c682-d8f9-4f11-9e77-647d4f14fd38)

**Comment by sensibleworld** [comment]
@ricardo17coelho You're right! Sorry about that. We did a BIG batch of organizing requests recently, and this one was mislabeled.

**Comment by sensibleworld** [comment]
These are now available in FA 7 Pro:
https://fontawesome.com/icons/folder-arrow-left?f=classic&s=solid
https://fontawesome.com/icons/folder-arrow-right?f=classic&s=solid
